pom.xml

```diff
@@ -54,10 +54,13 @@
 
 	<build>
 		<plugins>
-			<plugin>
-				<groupId>org.springframework.boot</groupId>
-				<artifactId>spring-boot-maven-plugin</artifactId>
-			</plugin>
+				<plugin>
```

Comment:
Please remove, as this will cause conflict with #1, or change according to my comment: https://github.com/wedding-hall-adm/wedding-management-service/pull/1#discussion_r505793348